ci-operator/step-registry/telcov10n/pre-ga-catalog/telcov10n-pre-ga-catalog-commands.sh

```
@@ -188,53 +190,63 @@ function findout_manifest_digest {
+  local prega_pull_secret_dir="$(mktemp -d)"
+  local prega_pull_secret_path="${prega_pull_secret_dir}/config.json"
+
+  cat > "${prega_pull_secret_path}" <<EOF
+  {
+    "auths": {
+      "quay.io": {
+        "auth": "${prega_pull_secret}"
+      }
+    }
+  }
+  EOF
```

Shellcheck (0.11.0)

[error] Remove indentation before end token (or use <<- and indent with tabs).

(SC1039)